pkg/repository/repository_test.go

```diff
@@ -22,9 +22,11 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
not a big deal, but we usually call this import `kerrors` (like, `kubernetes errors ...`)

```
tests/helpers/client.go
21:     "k8s.io/apimachinery/pkg/api/errors"

pkg/repository/repository_test.go
25:     apierrors "k8s.io/apimachinery/pkg/api/errors"

pkg/repository/repository.go
21:     api_errors "k8s.io/apimachinery/pkg/api/errors"

pkg/controller/supplychain/reconciler_test.go
27:     kerrors "k8s.io/apimachinery/pkg/api/errors"

pkg/controller/supplychain/reconciler.go
23:     kerrors "k8s.io/apimachinery/pkg/api/errors"

pkg/controller/pipeline/reconciler.go
24:     kerrors "k8s.io/apimachinery/pkg/api/errors"

pkg/controller/pipeline/reconciler_test.go
27:     kerrors "k8s.io/apimachinery/pkg/api/errors"

pkg/controller/workload/reconciler.go
23:     kerrors "k8s.io/apimachinery/pkg/api/errors"

pkg/controller/workload/reconciler_test.go
27:     kerrors "k8s.io/apimachinery/pkg/api/errors"
```

I think `apierrors` is more "go"y, but it'd be not very consistent with the rest of the codebase